bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx70-de.html

```diff
@@ -0,0 +1,114 @@
+        <p>{{ _('Perfekt! Du hast die neueste Version von Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+      <div class="mzp-c-box-emphasis">
+        <img class="c-emphasis-box-logo" src="{{ static('img/logos/firefox/logo-monitor.svg') }}" alt="">
+        <h2 class="c-emphasis-box-title">
+          {{ _('Schon Firefox Monitor ausprobiert?') }}
+          <span class="show-fxa-supported-signed-in">{{ _('Monitor kommt mit deinem Firefox-Konto') }}</span>
+        </h2>
+        <p>{{ _('Überprüfe mit Firefox Monitor, ob deine persönlichen Informationen bei Datenleaks anderer Unternehmen gefährdet wurden – und lass dich automatischen warnen, wenn deine Daten von Datenleaks betroffen sind.') }}</p>
+
+        <div class="show-fxa-supported-signed-out">
+          {{ monitor_button(
+            entrypoint=_entrypoint,
+            cta_type='FxA-Monitor',
+            cta_position='Primary',
+            utm_campaign=_utm_campaign,
+            button_text=_('Für Monitor anmelden')
+          ) }}
+        </div>
+
+        <div class="show-fxa-supported-signed-in">
+          {{ monitor_button(
+            entrypoint=_entrypoint,
+            cta_type='FxA-Monitor',
+            cta_position='Primary',
+            utm_campaign=_utm_campaign,
+            button_text=_('Einloggen')
+          ) }}
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="content-extra">
+    <div class="mzp-l-content">
+      <div class="l-columns-two">
+        <div class="c-picto-block">
+          <div class="c-picto-block-image">
+            <img src="{{ static('img/firefox/glyphs/tracking-protection.svg') }}" alt="">
+          </div>
+
+          <h3 class="c-picto-block-title">{{ _('Lass über 2000 Arten von Online-Trackern einfach blockieren – standardmäßig.') }}</h3>
+          <div class="c-picto-block-body">
+            <p>{{ _('Alle Firefox Browser für Desktop und Mobile kommen mit standardmäßig eingestelltem verbesserten Tracking-Schutz.') }}</p>
+            <a class="mzp-c-cta-link show-firefox-desktop-70 js-open-about-protections" href="https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop/?utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}&entrypoint={{ _entrypoint }}">{{ _('Sehen, was Firefox für dich geblockt hat') }}</a>
+
+            <a class="mzp-c-cta-link show-firefox-desktop-old" href="https://support.mozilla.org/kb/update-firefox-latest-release/?utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}&entrypoint={{ _entrypoint }}">{{ _('Firefox Browser updaten') }}</a>
+
+            <div class="show-default">
+              {{ download_firefox(alt_copy=_('Lad dir den Firefox Browser herunter'), download_location='primary cta', button_color='mzp-c-button mzp-t-primary mzp-t-product') }}
```

Comment:
Should this be `download_location='secondary cta'`? (same for other locale-specific templates).